### PR TITLE
Add missing fields to the checksuite event

### PR DESCRIPTION
**Pull request by prnvbn**
fixes https://github.com/google/go-github/issues/3277

### CODE REVIEW

File: github/checks.go [hunk 1]
```diff
@@ -85,7 +85,10 @@ type CheckSuite struct {
 	PullRequests []*PullRequest `json:"pull_requests,omitempty"`
 
 	// The following fields are only populated by Webhook events.
-	HeadCommit *Commit `json:"head_commit,omitempty"`
+	HeadCommit          *Commit `json:"head_commit,omitempty"`
+	LatstCheckRunsCount *int64  `json:"latest_check_runs_count,omitempty"`
```
Comment (gmlewis) [inline github/checks.go:89]:
```suggestion
	LatestCheckRunsCount *int64  `json:"latest_check_runs_count,omitempty"`
```

Comment (prnvbn) [reply github/checks.go]: done!

Comment (prnvbn) [reply github/checks.go]: feel free to resolve if you agree 🙃

**Review by gmlewis — APPROVED** [review]
Thank you, @prnvbn !
LGTM 
Merging.

**Pull request merged**